app.js

```diff
@@ -0,0 +1,32 @@
+const endpointJson = require("./endpoints.json");
+const express = require('express');
+const app = express();
+const {
+    fetchTopics
+} = require("./controllers/controller")
+
+app.use(express.json());
+
+
+
+app.get("/api", (req, res) => {
+    res.status(200).send({endpoints: endpointJson})
+});
+
+app.get('/api/topics', fetchTopics)
+
+
+app.use((err, req, res, next) => {
+    if (err.code === "22P02"){
+        res.status(400).send({msg: '404: invalid URL, bad request'})
+    }
```

Comment:
None of your tests are needing this yet, this may have been something you add in a future task.